### PR TITLE
fix: fix return of nodes

### PR DESCRIPTION
**Pull request by adevnadia**
appendChild, removeChild, insertBefore now return proper node instead of 'undefined'.

### CODE REVIEW

**Comment by adevnadia** [comment]
@treshugart @joscha @jpnelson take a look plz

File: test/unit/dom/dom.js [hunk 1]
```diff
@@ -983,4 +983,56 @@ describe('skatejs-named-slots dom', function () {
       expect(elem.parentNode).to.equal(frag);
     });
   });
+
+  describe('Return elements', function () {
```
Comment (treshugart) [inline test/unit/dom/dom.js]: Do you think it might be better to test this in each method's respective tests instead?

**Comment by treshugart** [comment]
It seems like there's a lot of returns that have to happen pretty deep down the callstack to get this to work. Would it be simpler if we returned what the user passed in from the public API? That seems to be what the standard DOM methods do anyways.

File: test/unit/dom/dom.js (hunk 1 above)
Comment (adevnadia) [inline test/unit/dom/dom.js]: You're absolutely right, sorry about this.
Done.

**Comment by adevnadia** [comment]
@treshugart updated returns (and tests), could you plz take another look?

**Comment by joscha** [comment]
LGTM

File: src/index.js [hunk 2]
```diff
@@ -782,19 +785,19 @@ const members = {
           return this.__removeChild(refNode);
         }
 
-        return removeNodeFromNode(this, refNode);
+        return removeNodeFromNode(this, refNode) || refNode;
```
Comment (treshugart) [inline src/index.js]: Maybe making these into `else if` blocks and then having a single `return refNode` at the bottom would remove some of the duplication?

**Comment by treshugart** [comment]
LGTM pending last comment.

**Pull request merged**